serverlog.go

```diff
@@ -145,7 +145,7 @@ type logItem struct {
 
 // listen is the listener which runs in its own gorutine and logs messages
 func listen() {
-	currentLogPath := path.Join(logDir, time.Now().Format("02-01-2006.crsv.log"))
+	currentLogPath := path.Join(logDir, time.Now().Format("2016-02-27.crsv.log"))
```

Comment:
There is a specific reason i chose the date 02/01/2006
take a look at https://gobyexample.com/time-formatting-parsing